pints/plot.py

```diff
@@ -8,6 +8,201 @@
 #
 from __future__ import division
 
+def log_likelihood_1d(log_likelihood, parameters, boundaries=None, n_eva=20, chain=None):
+    """
+    Takes a set of parameters as input and creates and returns a plot showing 
+    the log-likelihood function of each parameter with all the other 
+    parameters held at the given set of parameters.
+    
+    Arguments:
+
+    `log_likelihood`
+        A class (or that extends from) :class:`pints.LogLikelihood` of the 
```

Comment:
- The word "class" means "type". The argument `log_likelihood` is an object of a certain type. So you're not passing in a class, you're passing in an object of that class.*
- The general terminology here is "A extends B", no "from" is required! (It actually changes the meaning a bit: a branch extends from a tree, but a mathematical method can extend a previous method (no from!) meaning it encompasses the previous method _and_ adds something new. The second meaning is the one used when you say one class extends another)
- It's part of the rules of classes that if something accepts X, it accepts all subclasses Y that extend X. So you don't need to spell this out here!

\* This is extra important because in Python you can actually pass in a class, for example:
```
def creator(some_class):
    return some_class()
class Test(object):
    def __str__(self):
        return 'I am a test object'
x = creator(Test)
print(x)
I am a test object
```